src/core_plugins/tile_map/public/base_maps_visualization.js

```diff
@@ -116,6 +116,8 @@ export function BaseMapsVisualizationProvider(serviceSettings) {
         this._baseLayerDirty = true;
       });
       await this._updateBaseLayer();
+
+      return options;
```

Comment:
Why was this added? Looks like no one uses the return of this function

Reply:
sorry i have to remove this